features/gba-dev/uitsluiting-kiesrecht.feature

```diff
@@ -0,0 +1,20 @@
+# language: nl
+
+Functionaliteit: Uitsluiting kiesrecht
+
+
+	@gba
+	Abstract Scenario: Geen aanduiding uitgesloten kiesrecht (38.10) wel een einddatum uitsluiting kiesrecht
```

Comment:
willen we de @gba scenario's ook end-to-end testen?

Reply:
goede vraag, vroeg ik me ook al af. Ik heb het nu gemaakt zoals het in andere features ook is gedaan, dus niet end-to-end. Ik verwacht dus een datum in gbaDatum formaat. Omzetten naar end-2-end versie is m.i. aparte actie, waar @melsk-r en @JohanBoer mee aan de slag gaan